src/app/shared/components/shared-campus-event-list/shared-campus-event-list.component.html

```diff
@@ -0,0 +1,53 @@
+<ng-container *ngIf="eventListType === checkEventListType.ADMIN">
+    <div class="float-left">
+        <button mat-raised-button color="primary" (click)="navigateToProgramDetail()">查看项目详情</button>
+      </div>
+    <div class="float-right">
+        <button mat-raised-button color="primary" (click)="navigateToCreateForm()">创建新活动</button>
+    </div>
+</ng-container>
+
+<div class="loading-shade" *ngIf="isLoadingResults">
+  <mat-spinner></mat-spinner>
+</div>
+<div class="table-responsive">
+  <table class="table table-hover text-center">
+    <thead class="text-primary">
+      <tr>
+        <th scope="col">名字</th>
+        <th scope="col">时间</th>
+        <th scope="col">地点</th>
+        <th scope="col">总时长</th>
+        <th scope="col">参与者人数</th>
+        <ng-container *ngIf="eventListType === checkEventListType.USER">
+          <th scope="col">状态</th>
+        </ng-container>
+      </tr>
+    </thead>
+    <tbody>
+      <tr *ngFor="let item of results; let i = index">
+        <td scope="col" (click)="navigateToDetail(item)">{{ item.name }}</td>
```

Comment:
Why do you listen events for `(click)` multiple times, move it to `<tr>` so you just have to bind once.